.github/CODEOWNERS

```
@@ -8,7 +8,7 @@
 #
 #####################################################
 
-*.md @signalfx/docs
-*.rst @signalfx/docs
-docs/ @signalfx/docs
-README* @signalfx/docs @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
+*.md @signalfx/docs @signalfx/gdi-data-collection-approvers
+*.rst @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/gdi-data-collection-maintainers
+docs/ @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/gdi-data-collection-maintainers
+README* @signalfx/docs @signalfx/gdi-data-collection-approvers @signalfx/gdi-data-collection-maintainers
```

Sry, meant in general and not for just markdown files.

```suggestion
*.rst @signalfx/docs @signalfx/gdi-data-collection-approvers
docs/ @signalfx/docs @signalfx/gdi-data-collection-approvers
README* @signalfx/docs @signalfx/gdi-data-collection-approvers
```